js.go

```diff
@@ -866,12 +876,13 @@ func (m *Msg) Nak() error {
 	return m.ackReply(AckNak, false)
 }
 
-// Term this message from ever being delivered regardless of MaxDeliverCount.
+// MsgMetaData is metadata associated with received messages.
 func (m *Msg) Term() error {
 	return m.ackReply(AckTerm, false)
 }
 
-// Indicate that this message is being worked on and reset redelkivery timer in the server.
+// InProgress indicates that this message is being worked on
+// and reset the redelivery timer in the server.
 func (m *Msg) InProgress() error {
 	return m.ackReply(AckProgress, false)
```

Comment:
We could issue lots of these, the suppression code would suppress so we may need to look at ack type and the .ackd var.

Reply:
Updated to skip suppressing acks in case they are to progress report.